build/docker/zap-baseline.py

```diff
@@ -314,7 +314,8 @@ def main(argv):
                 time.sleep(5)
                 logging.debug('Delay passive scan check ' + str(delay - (datetime.now() - start_scan).seconds) + ' seconds')
 
-        zap_wait_for_passive_scan(zap)
+        # Allow twice as long as the time taken to spider for the passive scanning
+        zap_wait_for_passive_scan(zap, mins * 60 * 2)
```

Comment:
Maybe this is too short - I've checked on our daily scans and for some sites is can regularly take 10-20 mins for the passive scan to complete. Note that the box is being used for other things. Suggestions for a suitable multiplyer or should this be another param to the baseline script?